### ISSUE TITLE
@belapsed expression doesn't work on Julia 0.7

### ISSUE THREAD

**Issue by giordano** [JuliaCI/BenchmarkTools.jl#59]
On Julia 0.7 `@belapsed expression`, without keyword arguments, doesn't work:
```julia
julia> @belapsed sin(1)
ERROR: ArgumentError: tuple must be non-empty

julia> @belapsed sin(1) evals = 3
1.2999999999999999e-11
```
No problem with Julia 0.5.

Note that the first one isn't tested in `test/ExecutionTests.jl`.

BTW, isn't `1.3e-11` a bit too low as elapsed time?  On Julia 0.5, same machine (a laptop equipped with an Intel i7 CPU) time is of the order 1.2e-8, which looks more reasonable.

**Comment by giordano** [comment]
I'm starting to think that `@belapsed` is a bit broken on Julia 0.7:
```julia
julia> @belapsed sleep(0.5) evals = 3
1.2999999999999999e-11

julia> @btime sleep(0.5) evals = 3
  501.583 ms (5 allocations: 192 bytes)
```

**Comment by giordano** [comment]
`@belapsed` was probably broken by https://github.com/JuliaLang/julia/pull/21746.  I'm not sure how to fix this in a backward compatible way.

**Comment by StefanKarpinski** [comment]
> I'm not sure how to fix this in a backward compatible way.

The nuclear option is a conditional definition based on `VERSION`.

**Comment by jrevels** [comment]
closed by #77
